src/components/filterbox.pcss

```diff
@@ -28,17 +42,23 @@
 
   .search {
     background: none;
-    border: 0;
-    border-top: 1px solid rgb(201, 201, 201);
+    border: 1px solid #d9d9d9;
     box-sizing: border-box;
     display: inline-block;
+    font-family: 'Source sans pro', sans-serif;
     font-size: 1.5rem;
     height: 2em;
     left: 0;
     outline: none;
+    padding: 0.5em;
```

Comment:
We're mixing em and rem, I think we should stick to one (em's are a PITA to work with in my opinion :))

Reply:
Ah, It's not PITA to work with. However, I was doing it wrong in this case. Thanks for pointing that out.

We should make sure we are consistent with the use of rem em in the project at some point. If it's too messy we could just use rems everywhere I guess.